### DISCUSSION TITLE
File match - Ignore case

### DISCUSSION THREAD

**Discussion by marzoni-nicolas** [necojackarc/auto-request-review#101] [Q&A]
Hi team, I am wondering if there a way to ignore case when evaluating file names? 

Based on the documentation you are using [minimatch](https://github.com/isaacs/minimatch) and seems that we could disable the case sensitive using [nocase](https://github.com/isaacs/minimatch#nocase) feature. is there a way to enable/disable this from the configuration file?

Best,

**Comment by necojackarc** [comment]
@marzoni-nicolas Hi! Thanks for raising a question 😃 
As of today, it uses minimatch without passing any additional parameters, so it supports only the default behaviour:

https://github.com/necojackarc/auto-request-review/blob/6a51cebffe2c084705d9a7b394abd802e0119633/src/reviewer.js#L53-L60

If we'd like to pass parameters, we need to pass it through the Actions configuration file, and tweak this code a little bit.
If you can think of a good way to define that parameter in the config file, it shouldn't be that difficult to implement, I suppose.
